docs/overview/about.md

```diff
@@ -8,17 +8,17 @@ description: A self-sovereign blockchain network that allows developers to expre
 
 Archway is an incentivized smart contract platform that rewards developers — ushering in next gen dApps.
 
-The protocol gives developers the tools to easily build and launch scalable cross-chain dApps, and get rewarded for the value they contribute to the network.
+The protocol gives developers the tools to quickly build and launch scalable cross-chain dApps and get rewarded for the value the dApps contribute to the network.
 
 ## The Archway Mission
 
 The overarching goal of Archway is to foster an array of diverse and sustainable applications by empowering and rewarding developers that build on the network with baked in incentives and rewards.
 
-The internet helped to democratize access to information, but it hasn’t fully democratized access to capital and value. Entrepreneurs and developers around the world lack access to the same opportunities as their counterparts in more developed nations.
+The internet helped democratize access to information, but it hasn't fully democratized access to capital and value. Entrepreneurs and developers worldwide lack access to the same opportunities as their counterparts in more developed nations.
 
 Archway aims to level the playing field. Built into the protocol itself, Archway rewards developers based on the value and impact of their dApp, rather than their close association or connections to capital. 
 
-Unlike existing Web 2.0 platforms or early versions of Web3 layer 1s, where tokens are mostly concentrated in the hands of the first few early participants, Archway is designed to be shared across all contributors and stakeholders. The entrepreneurs and developers who are building on Archway get a true stake in the growth and governance of the protocol itself.
+Unlike existing Web 2.0 platforms or early versions of Web 3 Layer-1s, where tokens are mostly concentrated in the hands of the first few early participants, Archway is designed to be shared across all contributors and stakeholders. The entrepreneurs and developers who are building on Archway get a true stake in the growth and governance of the protocol itself.
```

Comment:
we're using the [Cosmos Word List](https://docs.google.com/spreadsheets/d/15raX8lfzzeeifGrDiXSFTHaEDbsMVlFJzKgnNelihjs/edit#gid=0) for spelling and convention